src/muya/lib/utils/url.js

```diff
@@ -1,8 +1,22 @@
 import { isValidAttribute } from '../utils/dompurify'
+import { isWin } from '../config' // __MARKTEXT_PATCH__
+import { hasMarkdownExtension } from './markdownFile' // __MARKTEXT_PATCH__
 
 export const sanitizeHyperlink = rawLink => {
-  if (rawLink && typeof rawLink === 'string' && isValidAttribute('a', 'href', rawLink)) {
-    return rawLink
+  if (rawLink && typeof rawLink === 'string') {
+    if (isValidAttribute('a', 'href', rawLink)) {
+      return rawLink
+    }
+
+    // __MARKTEXT_PATCH__
+    if (isWin && /^[a-zA-Z]:[/\\].+/.test(rawLink) && hasMarkdownExtension(rawLink)) {
+      // Create and try UNC path on Windows because "C:\file.md" isn't allowed.
+      const uncPath = `\\\\?\\${rawLink}`
```

Comment:
We add `\\\\?\\` prefix, can we still open the link?

Reply:
`\\?\` is a local  UNC path that is supported by Windows to allow paths with a length larger 255 and to connect network drives from external servers/IPs. I'm unsure whether this will for for network drives but it will for the common cases.